autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/rtc_auto_mode_manager/rtc_auto_mode_manager.param.yaml

```diff
@@ -7,8 +7,6 @@
       - "intersection"
       - "no_stopping_area"
       - "traffic_light"
-      - "ext_request_lane_change_left"
-      - "ext_request_lane_change_right"
       - "lane_change_left"
```

Comment:
@zulfaqar-azmi-t4 @rej55 
If this change does not look good to you, please make a PR to **awf/autoware_launch:main**.